static/swagger-specs/flow-service.yaml

```diff
@@ -920,41 +1410,1070 @@ paths:
+        schema:
+          type: string
+      - name: x-sandbox-name
+        in: header
+        description: All resources in Experience Platform are isolated to specific virtual sandboxes. Requests to Platform APIs require a header that specifies the name of the sandbox in which the operation will take place.
+        schema:
+          type: string
+      - name: RUN_ID
+        in: path
+        description: The ID of the run you want to retrieve.
+        required: true
+        schema:
+          type: string
+      responses:
+        "200":
+          description: Successful operation. A successful response returns details of specified run in JSON format.
+          x-summary: "Run Details Retrieved"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FlowRunsListResponse'
+        "403":
+          description: The OAuth token is missing.
+          x-summary: "Authorization Error - Missing OAuth Token"
+          content: {}
+        "503":
+          description: The service is unavailable.
+          x-summary: "Server Error - Service Unavailable"
+          content: {}
+    put:
+      tags:
+      - Runs
+      summary: Update a new flow run
+      description: Use this endpoint to update a flow run iteration.
+      operationId: updateRun
+      parameters:
+      - name: Authorization
+        in: header
+        description: "The access token which can be copied from your Experience Platform integration, prefixed with 'Bearer '. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
+        required: true
+        schema:
+          type: string
+      - name: x-api-key
+        in: header
+        description: "The API key which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
+        required: true
+        schema:
+          type: string
+      - name: x-gw-ims-org-id
+        in: header
+        description: "The organization ID which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
+        required: true
+        schema:
+          type: string
+      - name: x-sandbox-name
+        in: header
+        description: "All resources in Experience Platform are isolated to specific virtual sandboxes. Requests to Platform APIs require a header that specifies the name of the sandbox in which the operation will take place."
+        schema:
+          type: string
+      - name: If-Match
+        in: header
+        description: If-Match
+        required: true
+        schema:
+          type: string
+      - name: RUN_ID
+        in: path
+        description: Flow Run Id
```

Comment:
Same here - I'd go with something like "The ID of the flow run you want to update." 

I know it seems silly, but adding a little bit like that makes the docs feel more... alive? Robust? Less empty? Iunno. I think you get what I mean though LOL